### PR TITLE
doc: add missing historical release notes

### PR DESCRIPTION
**Pull request by fanquake**
0.20.2 and 0.21.2 are missing from master.

### CODE REVIEW

File: doc/release-notes/release-notes-0.20.2.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+- - #22124 Update translations after closing 0.20.x on Transifex (hebasto)
+- - #21471 fix bech32_encode calls in gen_key_io_test_vectors.py (sipa)
+- - #22837 mention bech32m/BIP350 in doc/descriptors.md (sipa)
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release:
+
+- - Aaron Clauson
```
Comment (jarolrod) [inline doc/release-notes/release-notes-0.20.2.md:167]: zero Diff for 21.2 but the 20.2 release notes being added here have formatting changes differing from https://raw.githubusercontent.com/bitcoin/bitcoin/v0.20.2/doc/release-notes.md

for example, why the double dashes here?

Comment (fanquake) [reply doc/release-notes/release-notes-0.20.2.md:167]: Copied from: https://lists.linuxfoundation.org/pipermail/bitcoin-core-dev/2021-October/000106.html. Can update.

**Comment by luke-jr** [comment]
Seems kind of a waste to include historical release notes in the repo/tarballs. Maybe the directory should just get wiped.

**Comment by fanquake** [comment]
I don't mind either way. It just doesn't make sense to leave the dir incomplete if it's going to remain.

**Review by jarolrod — APPROVED** [review]
ACK cb075d245e3a9106783f5edd0ed2cf1f7e43cdf8

Let's keep the historical docs for this release cycle, and revisit the conversations to remove them later. I assume this will get backported.

**Pull request merged**